ebu_tt_live/bindings/__init__.py

```diff
@@ -825,6 +843,75 @@ def content_to_string(self, begin=None, end=None):
 
 raw.br_type._SetSupersedingClass(br_type)
 
+class style_attr(IDMixin, RegionedElementMixin, LiveStyledElementMixin,
+               SemanticValidationMixin, raw.source_style, raw.element_style)
+    def _source_style(self, dataset):
```

Comment:
If I only need one should be fine to delete one, however I used the one for `div_type` as a template, and I'm thinking the difference is `source_style` grabs it from the document being tested, and `__source_style_copy__` makes sure that it's in the local instance.

At least, that's how I logicked it. Not sure if it's correct logic.